client/src/args.rs

```diff
@@ -0,0 +1,20 @@
+use anyhow::Result;
+use clap::Parser;
+
+use crate::config::Config;
+
+/// Simple program to greet a person
+#[derive(Parser, Debug)]
+#[command(version, about, long_about = None)]
+pub struct Args {
+    /// Path to a TOML configuration file.
+    #[arg(short, long, value_parser = parse_config_from_file, default_value = "config.toml")]
+    pub config: Config,
+}
+
+pub fn parse_config_from_file(path: &str) -> Result<Config> {
+    let config_file = std::fs::read_to_string(path)
+        .expect("Configuration file config.toml should be located in root directory.");
+    let config = toml::from_str(&config_file)?;
+    Ok(config)
+}
```

Comment:
Maybe we can return errors here instead of the main function. `clap` should have some nice formatting for this.